### PR TITLE
 tests created for the  Ethereum  Blockchain node providers

### PR DESCRIPTION
**Pull request by Philip-21**
This is pr fixes for #7 

@nguyer, @awrichar 
New Unit tests have been written for the ethereum blockchain node providers, the [Besu ](https://github.com/hyperledger/firefly-cli/blob/main/internal/blockchain/ethereum/besu/genesis.go)  and [geth](https://github.com/hyperledger/firefly-cli/tree/main/internal/blockchain/ethereum/geth) in the Ethereum Blockchain. 
These tests  pass successfully

### CODE REVIEW

File: internal/blockchain/ethereum/besu/besu_provider_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		Ctx   context.Context
+		Stack *types.Stack
+	}{
+		{
+			Name: "testcase1",
+			Ctx:  ctx,
+			Stack: &types.Stack{
+				Name:                   "TestBesuProviderWithEthconnect",
+				Members:                []*types.Organization{{OrgName: "Org1"}},
+				BlockchainProvider:     fftypes.FFEnumValue("BlockchainProvider", "EthConnect"),
```
Comment (nguyer) [inline internal/blockchain/ethereum/besu/besu_provider_test.go:25]: Should this be `EthConnect` here? `EthConnect` is a `BlockchainConnector` not a `BlockchainProvider`

File: internal/blockchain/ethereum/besu/besu_provider_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+				BlockchainConnector:    fftypes.FFEnumValue("BlockchainConnector", "EthConnect"),
+				BlockchainNodeProvider: fftypes.FFEnumValue("BlockchainNodeProvider", "EthConnect"),
```
Comment (nguyer) [inline internal/blockchain/ethereum/besu/besu_provider_test.go:27]: Should this be `EthConnect` here? `EthConnect` is a `BlockchainConnector` not a `BlockchainNodeProvider`

File: internal/blockchain/ethereum/besu/besu_provider_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			},
+		},
+		{
+			Name: "testcase2",
+			Ctx:  ctx,
+			Stack: &types.Stack{
+				Members:                []*types.Organization{{OrgName: "Org2"}, {OrgName: "org4"}},
+				Name:                   "TestBesuProviderWithEvmconnect",
+				BlockchainProvider:     fftypes.FFEnumValue("BlockchainProvider", "Geth"),
```
Comment (nguyer) [inline internal/blockchain/ethereum/besu/besu_provider_test.go:36]: Should this be `Besu` here? I don't think this test case is doing anything with Geth?

File: internal/blockchain/ethereum/besu/besu_provider_test.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+				BlockchainConnector:    fftypes.FFEnumValue("BlockchainConnector", "EvmConnect"),
+				BlockchainNodeProvider: fftypes.FFEnumValue("BlockchainNodeProvider", "Geth"),
```
Comment (nguyer) [inline internal/blockchain/ethereum/besu/besu_provider_test.go:38]: Should this be `Besu` here? I don't think this test case is doing anything with Geth?

Comment (Philip-21) [reply internal/blockchain/ethereum/besu/besu_provider_test.go]: @nguyer thanks for pointing these out i have made adjustments to the testcases, and placed the appropriate values for each of field

**Comment by Philip-21** [comment]
@nguyer I have written some tests for the Blockchain Node providers, I made adjustments to the corrections you pointed out .

**Comment by nguyer** [comment]
Thanks! This looks great

**Review by nguyer — APPROVED** [review]

**Pull request merged**